ccd-definition/CaseEventToFields/CareSupervision/manageOrders/3a_hearingDetailsSection.json

```diff
@@ -0,0 +1,66 @@
+[
```

Comment:
NOTE: I'm going to rename the section 3a to 3, and 3 to 4, etc once it's merging and passed testing, to avoid continuous merge conficts